bin/thumbnail

```diff
@@ -391,10 +433,9 @@ function thumb_params() {
 			echo "Height ${origheight} x Width ${origwidth}"
 			if [[ $origheight -gt $HEIGHT && $origwidth -gt $WIDTH ]]; then
 				THUMB_PARAMS="${QUALITY} -thumbnail ${WIDTH}x${HEIGHT}+0+0^ -gravity center -crop ${WIDTH}x${HEIGHT}+0+0!"
-			elif [ $origheight -gt $HEIGHT ] || [ $origwidth -gt $WIDTH ]; then
-				THUMB_PARAMS="${QUALITY} -gravity center -crop ${WIDTH}x${HEIGHT}+0+0! -thumbnail ${WIDTH}x${HEIGHT}+0+0"
 			else
-				THUMB_PARAMS="${QUALITY}"
+				recalculate_requested_size_for_crop_down $origwidth $origheight
+				THUMB_PARAMS="${QUALITY} -gravity center -crop ${WIDTH}x${HEIGHT}+0+0 -thumbnail ${WIDTH}x${HEIGHT}+0+0"
```

Comment:
The imagemagick documentation isn't always clear but I wonder if the above is the same as
```
THUMB_PARAMS="${QUALITY} -gravity center -crop ${WIDTH}x${HEIGHT}+0+0> -thumbnail ${WIDTH}x${HEIGHT}+0+0"
```

I only replaced the `!` with `>`. From [documentation](http://www.imagemagick.org/script/command-line-processing.php#geometry) the `>` will only shrink an image with dimension(s) larger than the width and height provided.

It would be nice to be able to lean on imagemagick as much as possible to simplify the logic in the bash script.